lib/furoshiki/linux_app.rb

```diff
@@ -0,0 +1,45 @@
+require 'furoshiki/base_app'
+require 'plist'
```

Comment:
Rather than one-off it, entered an issue to apply it broadly and get Rubocop on this project since it's outside Shoes and lacking that. https://github.com/shoes/furoshiki/issues/31